Library/ViewModels/PledgeCreditCardViewModel.swift

```diff
@@ -83,10 +101,19 @@ public final class PledgeCreditCardViewModel: PledgeCreditCardViewModelInputs,
 
     self.selectButtonIsSelected = cardAndSelectedCard
       .map(==)
+
+    self.spacerIsHidden = cardTypeIsAvailable.negate()
+    self.selectButtonEnabled = cardTypeIsAvailable
+    self.unavailableCardLabelHidden = cardTypeIsAvailable
+    self.unavailableCardText = self.creditCardProperty.signal.skipNil()
+      .map { Strings.You_cant_use_this_credit_card_to_back_a_project_from_project_country(
+        project_country: $0.projectCountry ?? ""
+      )
+      }
   }
 
-  fileprivate let creditCardProperty = MutableProperty<GraphUserCreditCard.CreditCard?>(nil)
-  public func configureWith(value: GraphUserCreditCard.CreditCard) {
+  fileprivate let creditCardProperty = MutableProperty<PledgeCreditCardValue?>(nil)
```

Comment:
Can we update the name of this property to match the fact that we now configure with `PledgeCreditCardValue`? Maybe `pledgeCreditCardValueProperty`?